src/fparser/common/tests/test_readfortran.py

```diff
@@ -604,7 +611,8 @@ def test_multi_put_item(ignore_comments):
 end program test
 '''
 
-FULL_FREE_EXPECTED = ['program test',
+FULL_FREE_EXPECTED = [u'!> Unicode comment: ❤ ✓ ☂ ♞ ☯',
+                      'program test',
                       '  implicit none',
                       "  character, paramater :: nature = 'free format'",
```

Comment:
It should also be changed on line 609. I'm not sure how this passes as it is invalid Fortran.

Reply:
Done. It is because we are only testing the file reader not the parser.